### ISSUE TITLE
Are you considering yolov8 ?

### ISSUE THREAD

**Issue by monkeycc** [igor-yusupov/rusty-yolo#5]
Are you considering yolov8 ?

Yolov8:
Classification
Detection
Segmentation
KeyPoint

**Comment by igor-yusupov** [comment]
> Are you considering yolov8 ?
> 
> Yolov8: Classification Detection Segmentation KeyPoint

yes, I want to optimize the current version, then I think to add new features from yolov8.

**Comment by monkeycc** [comment]
wow

Looking forward to it very much
